client/scripts/models/ChainInfo.ts

```diff
@@ -39,6 +39,26 @@ class ChainInfo {
     );
   }
 
+  public async updateChainData(name: string, description: string,) {
+    try {
+      await $.post(`${app.serverUrl()}/updateChain`, {
+        'id': app.activeChainId(),
+        'name': name,
+        'description': description,
+        'jwt': app.login.jwt,
+      }).then((r) => {
+        const updatedChain: ChainInfo = r.result;
+        this.name = updatedChain.name;
+        this.description = updatedChain.description;
+      });
+    } catch (err) {
+      console.log('Failed to update featured tags');
+      throw new Error((err.responseJSON && err.responseJSON.error)
```

Comment:
Worth discussing, but if we're catching errors and then throwing new ones with the same error logs, what's the point of catching them?

Reply:
The point of catching them right now is to produce a user-readable error message popup. But we're not very consistent about it. I could see it being somewhat beneficial to get more consistent at error handling over time.